<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetaInfo</class>
 <widget class="QWidget" name="DetaInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>0</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images出行助手/交通.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_city">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(206, 255, 253);
font: 12pt &quot;Source Code Pro&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_category">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 23pt &quot;Source Code Pro&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_text">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>751</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(206, 255, 253);
font: 12pt &quot;Source Code Pro&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_background</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>label_city</zorder>
  <zorder>label_category</zorder>
  <zorder>label_text</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
